lib/testsuite/index.js

```diff
@@ -817,6 +819,33 @@ class TestSuite {
 
     return runnable.run(this.commandQueue);
   }
+
+  takeSnapshot({node, result}) {
+    if (this.settings.trace.enabled && isTraceable(node.fullName)) {
+      this.snapShotPath = Snapshots.getFileName({
+        testSuite: this.api.currentTest.module,
+        testCase: this.api.currentTest.name,
+        commandName: node.fullName,
+        traceSettings: this.settings.trace,
+        output_folder: this.settings.output_folder
+      });
```

Comment:
This could be a constant path.

Reply:
we can use uuid or timestamp here, but it thought this would be nicer